frontend-project/src/models/global.ts

```diff
@@ -1,13 +1,12 @@
 import { notification } from 'antd';
-import { ReactNode } from 'react';
 import { IconType } from 'antd/lib/notification';
 
 export interface GlobalModelState {
   collapsed: boolean;
 }
 
-export const openNotificationWithIcon = (message: IconType, description: ReactNode) => {
-  notification[message]({ message, description });
+export const openNotificationWithIcon = (message: IconType, title: string, description: string) => {
```

Comment:
nie prościej byłoby w tym wypadku przyjmować obiekt?

Reply:
może i prościej ale w ten sposób działa. Możesz założyć issue na ustandaryzowanie tego w całym kodzie (bo wykorzystywane jest w Feature, Case, Tag, Channel).

Reply:
@kuskoman: To zależy. Jeśli obiekt {mesage, title, description} to OK, ale jeśli chodzi o to aby przekazywać obiekt tego samego typu co przyjmuje notification to średnio, bo tam można przekazać ReactNode. To pozwala zbudować bardzo rozbudowane notyfikacje, ale z drugiej strony notyfikacje powinny być proste i czytelne.